app/views/devise/passwords/new.html.haml

```diff
@@ -1,10 +1,17 @@
-%h2 Forgot your password?
-= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
+.text-center
+  %h1.h1.mt-6 Forgot your password?
+
+.mt-8.card.card-body
   = render "devise/shared/error_messages", resource: resource
-  .field
-    = f.label :email
-    %br/
+  = form_for(resource, as: resource_name, url: password_path(resource_name), builder: TailwindFormBuilder, html: {method: :post, class: 'space-y-6 -mt-6'}) do |f|
```

Comment:
add text before the input field: 

"Enter the email address associated with your account and we'll send you a link to reset your password."